.vscode/settings.json

```diff
@@ -1,9 +1,14 @@
 {
-  "workbench.colorCustomizations": {
-    "titleBar.activeBackground": "#4459ff",
-    "titleBar.activeForeground": "#fff",
-    "titleBar.inactiveBackground": "#4459ff1a",
-    "titleBar.inactiveForeground": "#858b9a",
-    "activityBar.activeBorder": "#4459ff"
-  }
+  "tailwindCSS.experimental.configFile": {
+    "apps/storybook-react/tailwind.config.js": "packages/design-system-react/src/**",
+    "packages/design-system-twrnc-preset/tailwind.config.js": "packages/design-system-react-native/src/**"
+  },
+  "files.associations": {
+    "*.css": "tailwindcss"
+  },
```

Comment:
Tells VS Code to treat all .css files as Tailwind CSS files. This means VS Code will use the Tailwind CSS language mode instead of the default CSS language mode. We get full IntelliSense support for Tailwind-specific syntax like:
- @tailwind directives
- @apply rules
- theme() function
- Other Tailwind-specific at-rules and functions
We currently only use the @tailwind directives but it may become more useful in the future